### ISSUE TITLE
Some questions about the method

### ISSUE THREAD

**Issue by zihanzheng-sjtu** [AuthorityWang/VideoGS#3]
Hello, once again, I would like to express my gratitude for your excellent work! I have carefully read through your paper and reviewed the code, and I have a few questions. The paper mentions that motion estimation is first carried out using hash coding, followed by fine-tuning on a motion-affected Gaussian sphere. The residuals of the normalized attributes are then estimated for entropy and quantized. Based on this approach, it seems that the objects being compressed and transmitted should be the normalized residuals and the hash table. However, in the code, the objects that are saved and compressed are the full Gaussians for each frame. This leads to the following questions:

1. What is the role of motion estimation in this process, and how does hash coding differ from directly fixing the other attributes while only training the position?
2. Why is entropy estimation performed on the residuals rather than directly on the attributes? Alternatively, why are the residuals not compressed directly, and instead all attributes are compressed?

I hope I haven't missed any key points. Thank you in advance for your patient responses!

**Comment by AuthorityWang** [comment]
Hi, sorry for the late response. 
1. The role of motion estimation is to generate each frame's Gaussian and has no relationship with the compression. We first use the motion estimation along with the fine-tuning training to generate each frame's Gaussian, then use the Video Codec to compress the data. The hash encoding used in the motion estimation stage is for faster convergence (you can refer to the ablation image Fig10 in the paper). 
2. As the Video Codec performs compression to the residual data of adjacent two frames, we apply the entropy estimation on the residual of the attributes to get higher performance of the compression with the video codec. We once tested compression directly on the residual, the per-frame storage has no significant difference, but the overall storage will be much larger as you need to store the first frame's Gaussian independently).

**Comment by zihanzheng-sjtu** [comment]
Thank you for your patient reply, it really helps a lot.
